pages/[slug].js

```diff
@@ -0,0 +1,49 @@
+import { useRouter } from "next/router";
+import Head from "next/head";
+import Image from "next/image";
+import styled from "styled-components";
+import { StyledOutivityImage } from "@/components/OutivityImage/StyledOutivityImage";
+
+export default function OutivityDetail({ outivities }) {
+  const router = useRouter();
+  const { slug } = router.query;
+  const outivity = outivities.find((outivity) => outivity.slug === slug);
+
+  if (!outivity) {
+    return <p>Loading...</p>;
+  }
+
+  return (
+    <>
+      <Head>
+        <title>{outivity.title}</title>
+      </Head>
+      <main>
+        <article>
+          <ul>
+            <li>
+              <h2>{outivity.title}</h2>
+              <StyledOutivityImage
+                src={outivity.image}
+                alt={outivity.title}
+                width={300}
+                height={200}
+              />
+              <p>
+                <strong>Location: </strong> {outivity.area},
+                <StyledCountryName>{outivity.country}</StyledCountryName>
+              </p>
+
+              <p>
+                <strong>Description: </strong> {outivity.description}
+              </p>
+            </li>
+          </ul>
+        </article>
+      </main>
+    </>
+  );
+}
+const StyledCountryName = styled.a`
+  margin-left: 7px;
+`;
```

Comment:
Without an `href attribute`, this anchor  `<a>` element doesn't function as a link. 

➡️ A more appropriate choice would be a `<span>` or a styled text element like `<strong>` or `<em>` if you want to emphasize the country name 🤷‍♀️
```suggestion
const StyledCountryName = styled.span`
  margin-left: 7px;
`;
```

Reply:
We already realized that by ourselves and changed it to a span :)